src/Destructurama.ByIgnoring/ByIgnoring/DestructureByIgnoringPolicy.cs

```diff
@@ -35,7 +35,9 @@ public DestructureByIgnoringPolicy(params Expression<Func<TDestructure, object>>
             var runtimeProperties = _destructureType.GetRuntimeProperties();
 
             _propertiesToInclude = runtimeProperties
-                .Where(p=> p.CanRead)
+                .Where(p => p.CanRead)
+                .Where(p => !p.GetMethod.IsStatic)
+                .Where(p => p.GetIndexParameters().Length == 0)
```

Comment:
> System.Reflection.TargetParameterCountException: 'Parameter count mismatch.'
> System.InvalidCastException: 'Unable to cast object of type 'Serilog.Events.ScalarValue' to type 'Serilog.Events.StructureValue'.'

was before this change inside `PropertyNamesInExpressionsAreIgnoredWhenDestructuring`